src/sac_stac/config.json

```diff
@@ -220,84 +220,84 @@
               "common_name": "coastal",
               "units": "1",
               "dtype": "uint16",
-              "nodata": -9999
+              "nodata": 0
             },
             {
               "name": "sr_band2",
               "common_name": "blue",
               "units": "1",
               "dtype": "uint16",
-              "nodata": -9999
+              "nodata": 0
             },
             {
               "name": "sr_band3",
               "common_name": "green",
               "units": "1",
               "dtype": "uint16",
-              "nodata": -9999
+              "nodata": 0
             },
             {
               "name": "sr_band4",
               "common_name": "red",
               "units": "1",
               "dtype": "uint16",
-              "nodata": -9999
+              "nodata": 0
             },
             {
               "name": "sr_band5",
               "common_name": "nir08",
               "units": "1",
               "dtype": "uint16",
-              "nodata": -9999
+              "nodata": 0
             },
             {
               "name": "sr_band6",
               "common_name": "swir16",
               "units": "1",
               "dtype": "uint16",
-              "nodata": -9999
+              "nodata": 0
             },
             {
               "name": "sr_band7",
               "common_name": "swir22",
               "units": "1",
               "dtype": "uint16",
-              "nodata": -9999
+              "nodata": 0
             },
             {
               "name": "bt_band10",
               "common_name": "lwir11",
               "units": "1",
               "dtype": "uint16",
-              "nodata": -9999
+              "nodata": 0
             },
             {
               "name": "bt_band1",
               "common_name": "lwir12",
               "units": "1",
               "dtype": "uint16",
-              "nodata": -9999
+              "nodata": 0
             },
             {
               "name": "pixel_qa",
               "common_name": "pixel_qa",
               "units": "1",
               "dtype": "uint16",
-              "nodata": -9999
+              "nodata": 0
             },
             {
               "name": "radsat_qa",
               "common_name": "radsat_qa",
               "units": "1",
               "dtype": "uint16",
-              "nodata": -9999
+              "nodata": 0
             },
             {
               "name": "sr_aerosol",
               "common_name": "aerosol",
               "units": "1",
               "dtype": "uint16",
-              "nodata": -9999
+              "nodata": 0
```

Comment:
change to value of 1

Reply:
this is an aerosol one not a pixel_qa one just to confirm?

Reply:
correct this should remain as 0 apologies

Reply:
there should be 4 to change though